OracleManagementAgent/README.md

```diff
@@ -112,50 +134,105 @@ Users can provide custom shell script commands to execute before starting Manage
+
+1. mkdir: cannot create directory '/opt/oracle/bootstrap': Permission denied
+
+    * Ensure the mounted volume exists and is accessible by the user used to run the container
+    * Verify the user USERID and GROUPID used match the permissions set on the mounted bind volume
+    * Verify the mounted bind volume exists on the host
+
+1. Invalid argument: /opt/oracle/mgmtagent_secret/input.rsp
+
+    * Ensure the install key exists at the required location
+
+1. Management Agent registration failures due to old state files from a prior install
+<!-- markdownlint-disable MD013 -->
+<!-- markdownlint-disable MD046 -->
+    * Once a Management Agent instance is deregistered that instance must be shutdown and any associated state files must be removed from the filesystem. Starting a deregistered Management Agent instance again can result in unregistered agent failures. This situation can present itself when old state files from a prior installation are present on the filesystem and made available to a new Management Agent container deployment. Run the command given below on the host filesystem to perform the necessary cleanup on the bind mount location and perform the deployment again starting at the prerequisites step.
+
+    ```shell
+    # cleanup old files from prior installation
+    $ rm -rf /home/$USER/oracle/
+    ```
+<!-- markdownlint-enable MD046 -->
+<!-- markdownlint-enable MD013 -->
+
+## Helpful commands
 
 1. Starting a stopped Management Agent Container
 
     ```shell
+    # start container and agent
     $ docker start mgmtagent-container
     ```
 
 1. Stopping a running Management Agent Container
 
     ```shell
+    # stop container and agent
     $ docker stop mgmtagent-container
     ```
 
 1. Inspecting logs of Management Agent Container
 
     ```shell
+    # inspect container and agent install logs
     $ docker logs mgmtagent-container
     ```
 
-1. Cleanup volume using Docker Compose
+1. Gathering UID and GID of a user from the host environment
 
     ```shell
-    $ docker-compose down --volumes
+    # see creating a user account below
+    $ id -u <username> # prints UID of user
+    $ id -g <username> # prints GID of user
     ```
 
-1. Cleanup volume using Docker CLI
+1. Creating a named user account during image development (optional)
 
     ```shell
-    $ docker volume rm mgmtagent-volume
+    # Commands given below can use used to precreate a user account in container image
+
+    $ groupadd -g <numeric-gid-value> <desired-groupname> # create group with gid
+    # example:
+    $ groupadd -g 9100 agentadmingrp
+
+    $ useradd <desired-username> -u <numeric-uid-value> -g <numeric-gid-value> -m -s /bin/bash # create user with uid/gid
+    # example:
+    $ useradd agentadminusr -u 9200 -g 9100 -m -s /bin/bash
+
+    # Note: Add useradd/groupadd to Dockerfile prefixed with the RUN directive to create user during image development
```

Comment:
Please remove all the comments and reference the examples in the instruction and then provide actual usable examples. Your note should probably reference the correct line number for more accuracy.